conf/net/ext_service/googlemaps.json

```diff
@@ -0,0 +1,3 @@
+{
+	"api_key" : "AIzaSyBEkw4PXVv_bsAdUmrFwatEyS6xLw3Bd9c/"
```

Comment:
You want to check in a random API key here instead of a real API key. Otherwise, there's no point in checking in a sample. Since you don't use google maps for real, the tests should continue to pass.

On production, I will replace with a different key.